src/MezzioInstaller/OptionalPackages.php

```diff
@@ -568,21 +568,6 @@ private function cleanUp(): void
         $this->recursiveRmdir($this->projectRoot . '.github');
         $this->recursiveRmdir($this->projectRoot . 'test/MezzioInstallerTest');
         $this->recursiveRmdir($this->projectRoot . 'docs');
-
-        $this->preparePhpunitConfig();
-    }
-
-    /**
-     * Remove the MezzioInstaller exclusion from the phpunit configuration
-     *
-     * @codeCoverageIgnore
-     */
-    private function preparePhpunitConfig(): void
-    {
-        $phpunitConfigFile = $this->projectRoot . 'phpunit.xml.dist';
```

Comment:
Perhaps just remove `phpunit.xml.dist` from `export-ignore`?

Reply:
ok.

Reply:
@Ocramius I made the appropriate fix and squashed the history.